sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/simulator/_model_tools/_identity_manager.py

```diff
@@ -1,8 +1,10 @@
 # ---------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # ---------------------------------------------------------
+# pylint: disable=C0303
```

Comment:
```suggestion
```

Running a formatter should automatically resolve this

https://github.com/Azure/azure-sdk-for-python/blob/main/doc/eng_sys_checks.md#running-locally

Reply:
Updated :)